### PR TITLE
Add support for winui3 

### PR DESCRIPTION
**Pull request by Gitii**
Hi,

I had trouble using this package with my winui3 app.
It looks like `Microsoft.Windows.Apps.Test` doesn't support winui 3 apps (but only true uwp apps).
I added a workaround which works for me.

### CODE REVIEW

**Comment by Gitii** [comment]
I reimplemented by previous workaround and replaced the previous `UAPApp` usage with my own code.
It handels both uwp and winui apps, including support for arguments.
These will be passed as activation args.
Of course, command line activation can be done separatly (the "legacy" route) but that's a separate kind of activation.

File: src/Infra/Communication/ApplicationMananger.cs [hunk 1]
```diff
@@ -4,167 +4,199 @@
...
 using System.Threading.Tasks;
+using WinAppDriver.Infra.Helper;
+using WinAppDriver.Infra.ModernApp;
 
 namespace WinAppDriver.Infra.Communication
 {
-  public class ApplicationMananger : IApplicationManager
-  {
-    private UIObject GetTopLevelWindow(UIObject uiObject)
+    public class ApplicationMananger : IApplicationManager
```
Comment (licanhua) [inline src/Infra/Communication/ApplicationMananger.cs]: Thanks to contribute this. Can you reformat all the files. I guess I convert `tab` to 2 spaces while you used 4 spaces

File: src/Infra/Helper/UIObjectHelpers.cs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            else
+            {
+                foreach (var className in classNames)
+                {
+                    var currentTimeout = timeoutInMilliseconds;
+
+                    var windowCondition = UICondition.CreateFromClassName(className)
+                        .AndWith(UICondition.CreateFromName(topWindow.Name));
+
+                    while (currentTimeout > 0
```
Comment (licanhua) [inline src/Infra/Helper/UIObjectHelpers.cs]: I think you actually want:

```
while (currentTimeout > 0 && element == null)
{
   for (var className in classNames)
   {
     ...
   }
   if (element == null)
   {
      Thread.sleep(100);
   }
}
```

**Review by licanhua — APPROVED** [review]

**Comment by licanhua** [comment]
@Gitii Your change is released to https://github.com/licanhua/YWinAppDriver/releases/tag/v0.2.107.0

**Pull request merged**